package.json

```
@@ -21,7 +21,12 @@
     "prop-types": "^15.6.2",
     "react": "^16.6.3",
     "react-dom": "^16.6.3",
```

```suggestion
    "react-dom": "^16.8.6",
```
обновите два данных пакета и все заработает:)